jenkins-cucumber-test-collector/src/main/java/com/capitalone/dashboard/collector/CucumberJsonToTestResultTransformer.java

```diff
@@ -48,23 +44,21 @@
 
     private TestSuite parseFeatureAsTestSuite(JSONObject featureElement) {
```

Comment:
There are two "end-to-end" tests in here https://github.com/capitalone/Hygieia/tree/master/jenkins-cucumber-test-collector/src/test - should cover now.